TreeControlInteractor/package.json

```diff
@@ -1,50 +1,57 @@
 {
   "name": "dnn-tree-control-interactor",
-  "version": "0.1.4",
+  "version": "0.1.5",
   "description": "Renders are selectable stateful tree",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "set NODE_ENV=production&&webpack -p"
+    "eslint": "eslint ./src/**/*.jsx",
+    "prepublishOnly": "npm-run-all prepublishOnly:*",
+    "prepublishOnly:eslint": "eslint ./src/**/*.jsx",
+    "prepublishOnly:babel": "babel src -d lib --source-maps",
+    "prepublishOnly:copy-less": "cpy **/*.less ../lib/ --cwd=src --parents",
+    "prepublishOnly:copy-svg": "cpy **/*.svg ../lib/ --cwd=src --parents"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/dnnsoftware/Dnn.React.Common"
   },
+  "babel": {
+    "presets": [
+      "es2015",
+      "react"
+    ],
+    "plugins": [
+      "transform-object-assign",
+      "transform-object-rest-spread"
+    ]
+  },
   "devDependencies": {
+    "babel-cli": "^6.18.0",
     "babel-core": "6.24.0",
-    "babel-loader": "6.2.4",
-    "babel-plugin-react-remove-prop-types": "^3.0.0",
-    "babel-plugin-transform-class-properties": "^6.22.0",
-    "babel-plugin-transform-object-assign": "^6.8.0",
-    "babel-plugin-transform-object-rest-spread": "^6.22.0",
-    "babel-plugin-transform-react-remove-prop-types": "^0.2.11",
+    "babel-eslint": "^8.0.1",
+    "babel-plugin-transform-object-assign": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-es2015": "6.6.0",
     "babel-preset-react": "6.5.0",
-    "css-loader": "0.23.1",
+    "cpy-cli": "^1.0.1",
     "eslint": "2.10.2",
-    "eslint-config-dnn": "*",
-    "eslint-import-resolver-webpack": "^0.3.2",
-    "eslint-loader": "1.3.0",
-    "eslint-plugin-filenames": "^1.0.0",
-    "eslint-plugin-import": "^1.10.2",
-    "eslint-plugin-react": "5.1.1",
-    "eslint-plugin-spellcheck": "0.0.7",
-    "less": "2.6.1",
-    "less-loader": "2.2.2",
-    "prop-types": "^15.5.8",
-    "raw-loader": "^0.5.1",
-    "style-loader": "0.13.0",
-    "svg-url-loader": "^2.0.2",
-    "webpack": "1.13.1"
-  },
-  "peerDependencies": {
-    "lodash": "4.17.4",
-    "react-portal-tooltip": "^1.1.4"
+    "eslint-config-dnn": "^0.0.7",
+    "eslint-import-resolver-node": "^0.3.1",
+    "eslint-plugin-babel": "^4.1.2",
+    "eslint-plugin-import": "^2.8.0",
+    "npm-run-all": "^4.1.1"
   },
   "dependencies": {
-    "dnn-text-overflow-wrapper-new": "0.0.4",
-    "shortid": "^2.2.8"
+    "shortid": "^2.2.8",
+    "lodash": "4.17.4",
```

Comment:
Are we doing tree shaking in end apps? If not maybe we could import the individual features of lodash?